casbin/log/default_logger.py

```diff
@@ -23,9 +25,18 @@ def write(self, *v):
             s = ""
             for vv in v:
                 s = s + str(vv)
-            logging.info(s)
+            self.logger.info(s)
 
     def writef(self, fmt, *v):
         """formats according to a format specifier and logs the message."""
         if self.enable:
-            logging.info(fmt, *v)
+            self.logger.info(fmt, *v)
+
+    def set_logger(self, log_level=logging.INFO, handler=None):
+        self.logger.setLevel(log_level)
+        if handler is None:
+            handler = logging.StreamHandler()
+            fmt = "%(asctime)s - %(levelname)s - %(message)s"
+            formatter = logging.Formatter(fmt)
+            handler.setFormatter(formatter)
+        self.logger.addHandler(handler)
```

Comment:
There is no need to set_logger here.
In `__init__`, you can create a custom logger directly, so that it will not affect the global log configuration.